app/views/decidim/votings/votes/_nvotes.html.erb

```diff
@@ -0,0 +1,18 @@
+<script>
+  function getHMAC(auth_data, callback) {
+    $.ajax("<%= token_voting_vote_path(voting.id) %>")
+      .done(function(data) {
+        callback("khmac:///sha-256;"+data);
+      })
+      .fail(function(jqXHR) {
+        window.location.href=jqXHR.responseText
+      });
+  }
+</script>
+
+<div class="row">
+  <div class="column nvotes-iframe" style="height:600px;">
+    <a class="agoravoting-voting-booth" href="https://<%= voting.voting_domain_name %>/booth/<%= voting.voting_identifier %>/vote?seed=<%=current_user.id%>" data-authorization-funcname="getHMAC"><%= t '.loading_cabin' %></a>
```

Comment:
Please, remove the seed part in the querystring. It was used for an specific voting and should not be there.

Reply:
ok